.github/workflows/coverage.yaml

```diff
@@ -52,18 +52,18 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: '1.13'
+        go-version: '^1.16'
```

Comment:
TIL: "You can allow a newer patch level version with tilde (~) and newer minor or patch level version with caret (^)."